src/geostore.js

```diff
@@ -427,6 +427,139 @@
 
   };
 
+  GeoStore.prototype.intersects = function(geojson){
+    var args = Array.prototype.slice.call(arguments);
```

Comment:
This is almost an exact copy of the `GeoStore.prototype.contains` method, the only change being the name and the use of `intersects` on line 499.